docs/index.md

```diff
@@ -1,30 +1,25 @@
-#Welcome to Jaeger's documentation!
+<img align="right" src="images/jaeger-vector.svg" width=400>
+#Jaeger Documentation
 
-[Jaeger](http://uber.github.io/jaeger/) is Uber's distributed tracing
-system.
+Welcome to Jaeger's documentation portal! Below, you'll find information for beginners and experienced Jaeger users. 
 
-##Architecture
+If you can't find what you are looking for, or have an issue not covered here, we'd love to hear from you either on [Github](https://github.com/uber/jaeger/issues), or on our [Mailing List](https://groups.google.com/forum/#!forum/jaeger-tracing). 
 
-Read [UberEng](https://eng.uber.com/) blog post on [Evolving Distributed
-Tracing at Uber](https://eng.uber.com/distributed-tracing/).
+##About
+Jaeger, inspired by Google's [Dapper](https://research.google.com/pubs/pub36356.html), is a distributed tracing system which captures timing information on microservice architectures. Currently, this information is used for realtime profiling, and empirically determining service dependencies.
```

Comment:
similar to the press release, we need better paragraph about this, the general value of tracing.

At minimum this can be a short one like this:

> Jaeger is a distributed tracing system used to monitor, profile, and troubleshoot microservices.